### ISSUE TITLE
Clone the enterprise repo

### ISSUE THREAD

**Issue by guoyr** [mongodb/server-workflow-tool#1]

**Comment by markbenvenuto** [comment]
Are you going to symlink it (many developers, not me though) or clone it deep into the tree (i.e. evergreen)?

**Comment by guoyr** [comment]
Good question! I'll let @IanWhalen decide on the canonical way to set things up.

My 2¢ is to use a symlink as the default on macOS. If they want to use a different OS, they can configure their dev environment differently.

**Comment by IanWhalen** [comment]
oh you totally don't want me making the canonical decision.  sorry, will have to leave that to the engineers.  symlinking would also be my default though just because I like having the same directory structure locally as in github, i.e.

```
~/code/10gen/mongo-enterprise-modules
~/code/10gen/kernel-tools
~/code/mongodb/mongo
```
etc.

**Comment by milkie** [comment]
Note that symlinking actually makes it a bit harder to do development, as you lose the ability to run "scons lint" and have it identify lint problems in the enterprise module.  (Also, the duplicate-error-code checker is affected in the same way.)  So I would encourage the default as cloned directly in the tree.  Helpfully, this also mirrors what Evergreen does.

**Comment by IanWhalen** [comment]
that's a better rationale than mine.  so unless you've got another reason for favoring symlinks, robert, I would go with cloning into the tree.

**Comment by milkie** [comment]
Along the same lines, I would encourage that the ninja module also be cloned directly into the modules subdirectory (if that's being considered elsewhere as part of this project.)

**Comment by IanWhalen** [comment]
Fixed in 87ca47a269ba62e3a0e4dc09096772d420bec77b
